docker-entrypoint.sh

```diff
@@ -61,6 +61,11 @@ if [ $S3FS_DEBUG = "1" ]; then
     DEBUG_OPTS="-d -d"
 fi
 
+# Additional S3FS options
+if [ -n "$S3FS_ARGS" ]; then
+    S3FS_ARGS="-o $S3FS_ARGS"
```

Comment:
This only makes sense if the use provides one argument, e.g., `sigv2`.  If they provide two, then they need to add `-o` for all options after the second, e.g., `sigv2 -o nomultipart`.  This seems to be more confusing.

Reply:
According to documentation and my own testings, comma-separated options work fine.

Example: `-o use_path_request_style,allow_other,default_acl=public-read`

Reply:
Agreed -- I forgot about this syntax.

Reply:
I might later add a bulletpoint to the README to make the usage of `S3FS_ARGS` clear